.github/workflows/test-mutations.yml

```diff
@@ -78,3 +78,12 @@ jobs:
 
       - name: "Mutation Tests"
         run: "composer test:mutation -- --logger-github=false"
+
+      - name: "Store Infection logs"
+        if: success() || failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: infection-logs-pr-${{ github.event.pull_request.number }}
+          path: ./var/infection/*.log
+          overwrite: true
+          retention-days: 7
```

Comment:
if those logs are readable we can add them to job summary: 

```
      - name: "Set job summary"
        run: |
          cat ./var/infection/*.log >> $GITHUB_STEP_SUMMARY
```

Reply:
Main is... 7MB+ of data 😅 but summary is not best either, but after checking them I'm not sure it's worth holding them anyway

Reply:
oh shit, then maybe lets drop it 😅